packages/fuel-indexer-macros/src/native.rs

```diff
@@ -31,20 +31,30 @@ pub fn handler_block_native(
 /// indexer module, not within the scope of the entire lib module.
 fn native_prelude() -> proc_macro2::TokenStream {
     quote! {
-        use fuel_indexer_plugin::native::*;
-        use fuel_indexer_plugin::{{serialize, deserialize}, types::*, serde::{Deserialize, Serialize}, serde_json};
+        type B256 = [u8; 32];
+
+        static mut db: Option<Arc<Mutex<Database>>> = None;
 
         // TODO: Eventually prevent these types of implicity imports and have users import
         // all dependencies explicity (preferably through a single crate).
+        use fuel_indexer_utils::{
+            plugin::{
+                native::*,
+                serde::{Deserialize, Serialize},
+                types::*,
+            },
+            prelude::{
```

Comment:
@deekerno Any reason we're not importing through `plugin` here? Ideally, the `prelude` is only used by users. (Importing through the plugin makes things a bit more explicit for devs)